app/src/main/java/com/akshay/newsapp/news/ui/adapter/NewsArticlesAdapter.kt

```diff
@@ -45,13 +43,11 @@ class NewsArticlesAdapter(
             //TODO: need to format date
             //tvListItemDateTime.text = getFormattedDate(newsArticle.publishedAt)
             newsPublishedAt.text = newsArticle.publishedAt
-            Glide.with(context)
-                    .load(newsArticle.urlToImage)
-                    .apply(RequestOptions()
-                            .placeholder(R.drawable.tools_placeholder)
-                            .error(R.drawable.tools_placeholder)
-                            .diskCacheStrategy(DiskCacheStrategy.ALL))
-                    .into(newsImage)
+            newsImage.load(newsArticle.urlToImage) {
+                placeholder(R.drawable.tools_placeholder)
+                error(R.drawable.tools_placeholder)
+                // Disk cache is enabled by default in Coil load request
```

Comment:
I think we can skip the comment or move it on line 47. WDYT?

Reply:
Skipping the comment would be fine. We can customize the caching later if needed.